benchmarks/src/main/scala/zio/stm/TMapContentionBenchmarks.scala

```diff
@@ -29,8 +30,8 @@ class TMapContentionBenchmarks {
     val map          = unsafeRun(TMap.fromIterable(data).commit)
     val ref          = ZTRef.unsafeMake(data.toMap)
 
-    mapUpdates = UIO.foreachPar_(keysToUpdate)(i => map.put(i, i).commit.repeat(schedule))
-    refUpdates = UIO.foreachPar_(keysToUpdate)(i => ref.update(_.updated(i, i)).commit.repeat(schedule))
+    mapUpdates = ZIO.foreachPar_(keysToUpdate)(i => map.put(i, i).commit.repeat(schedule))
```

Comment:
This feels like a situation where we could potentially use a few repeat variants that don't use a Schedule to do very simple things like repeat an effect a specified number of times.

Reply:
We could also define a `timeless` combinator that deletes `Clock` from a requirement, so you could do: `timeless(effect.retry(foo))`.